bundle/jsky.app.ot/src/main/java/jsky/image/operator/CutLevelOpImage.java

```diff
@@ -154,8 +147,19 @@ protected void accumulateStatistics(String name, Raster source, Object ar) {
                 }
                 break;
 
+            case DataBuffer.TYPE_DOUBLE:
```

Comment:
I think you can just add this below the `TYPE_FLOAT` case instead of repeating it, unless you think it is clearer this way?

```java
case DataBuffer.TYPE_FLOAT:
case DataBuffer.TYPE_DOUBLE:
    {
        ...
    }

Reply:
Never mind, I guess it does call a different method in the end.  I didn't see that at first.

Reply:
Yeah … primitives, yay!